frontend/src/components/Exp.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="job-info-wrapper">
+    <input
+      type="text"
+      class="job-title"
+      v-model="experience.jobTitle"
+      placeholder="Job Title"
+      @input="saveExperience"
+    />
+
+    <input
+      type="text"
+      class="company"
+      v-model="experience.company"
+      placeholder="Company"
+      @input="saveExperience"
+    />
+
+    <input
+      type="text"
+      v-model="experience.location"
+      placeholder="Location"
+      @input="saveExperience"
+    />
+    <label>From</label>
+    <input
+      type="date"
+      v-model="experience.startingYear"
+      @input="saveExperience"
+    />
+
+    <label>To </label>
+    <input
+      type="date"
+      v-model="experience.endingYear"
+      @input="saveExperience"
+    />
+  </div>
+
+  <textarea
+    class="wdud"
+    type="text"
+    v-model="experience.whatYouDid"
+    placeholder="What did you do?"
+    @input="saveExperience"
+  />
+
+  <div>
```

Comment:
<img width="446" alt="Screenshot 2024-01-23 at 12 09 32" src="https://github.com/philippOkie/resume-builder-vue/assets/48657633/ce1270e8-8251-4d38-afd5-b9c800fdda4b">

Reply:
might need to double check the filename. **Readability counts**